webpack.config.js

```diff
@@ -1,5 +1,6 @@
 const ReactRefreshWebpackPlugin = require('@pmmmwh/react-refresh-webpack-plugin');
 const { sentryWebpackPlugin } = require('@sentry/webpack-plugin');
+const CircularDependencyPlugin = require('circular-dependency-plugin');
```

Comment:
remove here perhaps?

Reply:
I kept it but added a logic in order to add validation by switching a Boolean value